### ISSUE TITLE
.NET 5.0

### ISSUE THREAD

**Issue by Brodutch** [MicrosoftDocs/quantum-docs-pr#1156]
The prerequisite  is  listed as ".NET Core SDK 3.1 or later" but at least on Linux I could not compile with .NET 5.0. 

---
#### Document details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 97e509ca-7867-e8f0-45f1-bb20d27ad232
* Version Independent ID: a6a281f8-53e0-8bdb-2b6e-8228fc2204f2
* Content: [Develop with Q# applications in an IDE - Microsoft Quantum](https://docs.microsoft.com/en-ca/quantum/quickstarts/install-command-line?tabs=tabid-vscode)
* Content Source: [articles/quickstarts/install-command-line.md](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/live/articles/quickstarts/install-command-line.md)
* Product: **quantum**
* GitHub Login: @Bradben
* Microsoft Alias: **v-benbra**

**Comment by geduardo** [comment]
Hi @Brodutch, thank you for filling the issue. 

@cgranade, do you know if the QDK should be compatible with .NET 5.0?

**Comment by cgranade** [comment]
If I understand correctly, yes. @ricardo-espinoza may know more here.

**Comment by ricardo-espinoza** [comment]
Right. However, we had an issue not so long ago in which we needed to have .NET 3.1 installed as well.
@vxfield , do you know if that has been resolved already?

**Comment by Brodutch** [comment]
> Right. However, we had an issue not so long ago in which we needed to have .NET 3.1 installed as well.
> @vxfield , do you know if that has been resolved already?

That's what I ended up doing (I now have both 3.1 and 5.0).

**Comment by vxfield** [comment]
In most cases, QDK should work just with .NET SDK 5.0 and without .NET Core SDK 3.1.
The issue we found in November was specifically with the Language Server (that is used to enrich the experience of editing Q# files on Visual Studio and VS Code) and it was requiring .NET Core SDK 3.1 to work.

Details can be found on [Q# Compiler issue 737](https://github.com/microsoft/qsharp-compiler/issues/737) and [MS Build Locator issue 92](https://github.com/microsoft/MSBuildLocator/issues/92).

Since Nov 2021 release, and continuing with the Jan 2021 release, **we require .NET Core SDK 3.1** for the Language Server to work (and as a consequence to QDK in general).

We are working on a solution to remove the dependency of .NET Core SDK 3.1 but we need more time to test it and it won't make it for the Jan 2021 release (but likely will for Feb 2021).

**Comment by vxfield** [comment]
@geduardo, perhaps we should update the pre-req documentation to

".NET Core SDK 3.1" instead of ".NET Core SDK 3.1 or later" as when we wrote that, we didn't know that .NET SDK 5.0 would be release and not work for us.
